server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -369,25 +377,53 @@ static boolean isListOfStrings(List<Object> list) {
             context.put(k, v);
         }
 
-        // Added to build.yml in 1.2.174 (setting for using Clang 9)
+        // Added to build.yml in 1.2.185 (setting for using isystem headers)
```

Comment:
The upgrade workaround.
Since the iOS 14.5 has a different layout than the 14.0 version, and those build.yml's didn't have these settings.